identify-slow-queries.md

```diff
@@ -64,7 +64,13 @@ Slow query basics:
 * `Txn_start_ts`: The start timestamp and the unique ID of a transaction. You can use this value to search for the transaction-related logs.
 * `Is_internal`: Whether a SQL statement is TiDB internal. `true` indicates that a SQL statement is executed internally in TiDB and `false` indicates that a SQL statement is executed by the user.
 * `Index_names`: The index names used by the statement.
-* `Stats`: The health state of the involved tables. `pseudo` indicates that the state is unhealthy.
+* `Stats`: The health state, internal version, total row count, row count of modifications and load status of statistics that are used during this query. `pseudo` indicates that the state is unhealthy. If the optimizer tried to use some statistics that are not fully loaded, they will also be printed. For example, `t1:439478225786634241[105000;5000][col1:allEvicted][idx1:allEvicted]` means the following:
+    - `t1`: statistics on table `t1` are used during this query
+    - `439478225786634241`: its internal version
+    - `105000`: the total row count in the statistics
+    - `5000`: the number of rows of modification recorded since the last time statistics are collected
+    - `col1:allEvicted`: statistics on the column `col1` are not fully loaded
+    - `idx1:allEvicted`: statistics on the index `idx1` are not fully loaded
```

Comment:
```suggestion
    - `t1`: statistics on table `t1` are used during query optimization.
    - `439478225786634241`: the internal version.
    - `105000`: the total row count in the statistics.
    - `5000`: the number of rows modified since the last statistics collection.
    - `col1:allEvicted`: statistics on the column `col1` are not fully loaded.
    - `idx1:allEvicted`: statistics on the index `idx1` are not fully loaded.
```